### PR TITLE
Update SDK based on examples changes to swagger spec

### PR DESCRIPTION
**Pull request by nathannfan**
## Description
Swagger spec pull request: https://github.com/Azure/azure-rest-api-specs/pull/954

This change updated the auto-generated SDK based on changes to the swagger spec that were done in response to examples validation and Fiddler testing.
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as a link to the swagger spec, used to generate the code.
- [ ] The `project.json` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@nathannfan__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (nafan). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/ResourceManagement/SqlManagement/Sql.Tests/DatabaseRestoreScenarioTests.cs [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        [Fact]
+        public void TestDatabasePointInTimeRestore()
+        {
+            // Warning: This test takes around 20 minutes to run in record mode.
+
+            string testPrefix = "sqlrestoretest-";
+            string testName = this.GetType().FullName;
+            SqlManagementTestUtilities.RunTestInNewV12Server(testName, "TestDatabasePointInTimeRestore", testPrefix, (resClient, sqlClient, resourceGroup, server) =>
+            {
+                Dictionary<string, string> tags = new Dictionary<string, string>()
```
Comment (nathannfan) [inline src/ResourceManagement/SqlManagement/Sql.Tests/DatabaseRestoreScenarioTests.cs]: Remove this

File: src/ResourceManagement/SqlManagement/Sql.Tests/ElasticPoolActivityScenarioTests.cs [hunk 2]
```diff
@@ -99,5 +100,75 @@ public void TestListElasticPoolActivity()
...
+                    Location = server.Location,
+                    ElasticPoolName = epName2
+                };
+                sqlClient.Databases.CreateOrUpdate(resourceGroup.Name, server.Name, dbName, dbInput);
+
+                // Get the Elastic Pool Database Activity List for second pool
+                activity = sqlClient.ElasticPools.ListDatabaseActivity(resourceGroup.Name, server.Name, epName2);
+                Assert.Equal(2, activity.Where(a => a.DatabaseName == dbName).Count());
+                Assert.Equal(1, activity.Where(a => a.DatabaseName == dbName && a.Operation == "CREATE").Count());
+                Assert.Equal(1, activity.Where(a => a.DatabaseName == dbName && a.Operation == "UPDATE").Count());
```
Comment (jaredmoo) [inline src/ResourceManagement/SqlManagement/Sql.Tests/ElasticPoolActivityScenarioTests.cs]: Can you also test removing the database from the pool?

**Review by jaredmoo — APPROVED** [review]
Looks great, I would just like to see a test to remove db from pool.

**Comment by jaredmoo** [comment]
Approved

File: src/ResourceManagement/SqlManagement/Microsoft.Azure.Management.Sql/Generated/ElasticPoolsOperations.cs [hunk 3]
```diff
@@ -303,24 +305,6 @@ internal ElasticPoolsOperations(SqlManagementClient client)
                     throw new Microsoft.Rest.SerializationException("Unable to deserialize the response.", _responseContent, ex);
                 }
             }
-            // Deserialize Response
```
Comment (shahabhijeet) [inline src/ResourceManagement/SqlManagement/Microsoft.Azure.Management.Sql/Generated/ElasticPoolsOperations.cs]: @nathannfan did the swagger changed and is no longer returning 202?

File: src/ResourceManagement/SqlManagement/Microsoft.Azure.Management.Sql/Generated/ElasticPoolsOperations.cs [hunk 4]
```diff
@@ -1114,12 +1103,8 @@ internal ElasticPoolsOperations(SqlManagementClient client)
         /// <return>
         /// A response object containing the response body and response headers.
         /// </return>
-        public async System.Threading.Tasks.Task<Microsoft.Rest.Azure.AzureOperationResponse<System.Collections.Generic.IEnumerable<ElasticPoolDatabaseActivity>>> ListDatabaseActivityWithHttpMessagesAsync(string elasticPoolName, string resourceGroupName, string serverName, System.Collections.Generic.Dictionary<string, System.Collections.Generic.List<string>> customHeaders = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken))
+        public async System.Threading.Tasks.Task<Microsoft.Rest.Azure.AzureOperationResponse<System.Collections.Generic.IEnumerable<ElasticPoolDatabaseActivity>>> ListDatabaseActivityWithHttpMessagesAsync(string resourceGroupName, string serverName, string elasticPoolName, System.Collections.Generic.Dictionary<string, System.Collections.Generic.List<string>> customHeaders = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken))
```
Comment (shahabhijeet) [inline src/ResourceManagement/SqlManagement/Microsoft.Azure.Management.Sql/Generated/ElasticPoolsOperations.cs]: @nathannfan was name simplification used when the code was generated via AutoRest?

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
What version of AutoRest used to generate code.
Was the latest version used.
Also I don't see name simplification was used during code generation.

File: src/ResourceManagement/SqlManagement/Microsoft.Azure.Management.Sql/Generated/ElasticPoolsOperations.cs (hunk 3 above)
Comment (nathannfan) [inline src/ResourceManagement/SqlManagement/Microsoft.Azure.Management.Sql/Generated/ElasticPoolsOperations.cs]: Swagger still documents 202, but no longer has a response body, which I believe is why this section is removed.

File: src/ResourceManagement/SqlManagement/Microsoft.Azure.Management.Sql/Generated/ElasticPoolsOperations.cs (hunk 4 above)
Comment (nathannfan) [inline src/ResourceManagement/SqlManagement/Microsoft.Azure.Management.Sql/Generated/ElasticPoolsOperations.cs]: @shahabhijeet Not sure how that is specified.  Is there documentation?

**Comment by cormacpayne** [comment]
@azuresdkci test this please

File: src/ResourceManagement/SqlManagement/Microsoft.Azure.Management.Sql/Generated/ElasticPoolsOperations.cs (hunk 4 above)
Comment (nathannfan) [inline src/ResourceManagement/SqlManagement/Microsoft.Azure.Management.Sql/Generated/ElasticPoolsOperations.cs]: Synced up with Cormac.  This requires a later version of AutoRest.

**Comment by shahabhijeet** [comment]
@nathannfan starting next PR, please use the latest Autorest 1.0 to regen code. It's ok for this PR.

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**